plugins/scanners/fossid/src/main/kotlin/FossIdNamingProvider.kt

```diff
@@ -46,6 +47,9 @@ class FossIdNamingProvider(
     companion object {
         @JvmStatic
         val FORMATTER: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyyMMdd_HHmmss")
+
+        @JvmStatic
+        val MAX_SCAN_CODE_LEN = 254
```

Comment:
This should be `const` instead of `@JvmStatic`.

Reply:
OK